spec/aws-record/record_spec.rb

```diff
@@ -164,5 +164,61 @@ module Aws
       end
     end
 
+    describe 'inheritance support for table name' do
```

Comment:
We probably want some test cases here for when the default table names are used - which will require us to actually define `class ParentModel....` I think.

Reply:
Completed.